marchingsquares-isobands.js

```diff
@@ -71,6 +71,15 @@
   function isoBands(data, minV, bandwidth, options){
     var settings = {};
 
+    /* validation */
+    if (!data) throw new Error('data is required');
+    if (!Array.isArray(data) || !Array.isArray(data[0])) throw new Error('data should be an array of arrays');
+    if (minV === undefined || minV === null) throw new Error('lowerBand is required');
+    if (bandwidth === undefined || bandwidth === null) throw new Error('bandWidth is required');
+    if (isNaN(+minV)) throw new Error('lowerBand must be a number');
+    if (isNaN(+bandwidth)) throw new Error('bandWidth must be a number');
+    if (!!options && options.constructor !== 'object') throw new Error('options must be an object');
```

Comment:
This test fails, at least in web browser. `options.constructor` is a `function`, so I would suggest using `typeof optionst !=== 'object'`

Reply:
Oh, glad you got that, never tested in the browser 👍